typings/index.d.ts

```diff
@@ -0,0 +1,8 @@
+interface Options {
```

Comment:
Looks good to me!

Note that there actually is a return value in the source code (though it's not documented). It's a "restore" function which restores useEffect to its original definition.